index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Embed media - YouTube clone</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="https://kit.fontawesome.com/5036f34c07.js" crossorigin="anonymous"></script>
+
+
+</head>
+
+<body>
+    <header>
+        <nav class="homeBar">
+            <div class="menu">
+                <i class="fas fa-bars"></i>
+            </div>
+            <div class="logo">
+                <i class="fab fa-youtube"></i>
+                <h2>YouTube</h2>
+            </div>
+
+            <div class="searchLink">
+                <input type="text" name="contentInput" class="searchBar" placeholder="search">
+                <button class="butn"><i class="fas fa-search"></i></button>
+            </div>
+        
+            <div class="icons">
+                <i class="fas fa-video icon"></i>
+                <i class="fas fa-bell "></i>
+                <i class="fas fa-user icon"></i>
+
+            </div>
+
+        </nav>
+    </header>
+
+    <div class="container">
+        <main class="left-container">
+            <iframe width="100%" height="387" src="https://www.youtube.com/embed/SpeDK1TPbew" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+            <section class="userData">
+                <h2>The Rise of Open-Source Software</h2>
+
+                <div class="channel">
+
+                    <div class="channel1">
+                        <img src="images/propic.jpg" alt="profile picture">
+                    </div>
+
+                    <div class="channel2">
+                        <h3>Tech News Today</h3>
+                        <div class="subs">
+                            <button id="subscribe">SUBSCRIBE</button>
+                            <small class="small">3.5M</small>
+                        </div>
+                    </div>
+
+                    <div class="channel3">
+                        <h4>50,243,656 views</h4>
+                    </div>
+                </div>
+
+                <hr style=": blue;" />
+
+                <div class="share">
+                    <div class="share-left">
+
+                        <i class="fas fa-plus"><small>add</small></i>
+                        <i class="title-icon fas fa-share">share</i>
```

Comment:
- [ ] It needs to be a button:

![image](https://user-images.githubusercontent.com/61479648/90439011-3b787180-e0ab-11ea-936a-ec4157abc283.png)